src/__integrations__/pagopaApi.browser.integration.test.ts

```diff
@@ -0,0 +1,139 @@
+import { Server } from 'http';
+
+import { Browser, launch } from 'puppeteer';
+import { createHttpTerminator } from 'http-terminator';
+import stubServer from './stubServer';
+
+describe('Test the browser test environment', () => {
+  it('Puppeteer should work', async () => {
+    const myBrowser: Browser = await launch();
+    const page = await myBrowser.newPage();
+    const [result] = await Promise.all([
+      page.waitForResponse(response => response.url() === 'https://api.ratesapi.io/api/latest'),
+      page.evaluate(() =>
+        fetch('https://api.ratesapi.io/api/latest', {
+          headers: {},
+          method: 'GET',
+        }),
+      ),
+    ]);
+    await expect(result.json()).resolves.toHaveProperty('base');
+    await myBrowser.close();
+  });
+});
+
+describe('Custom Fetches test in browser environment', () => {
+  const PORT = 5000;
+  const HOST = 'localhost';
+
+  it('When the user navigate to the bad endpoint of the local server, it should respond 404', async () => {
```

Comment:
https://pagopa.github.io/io-docs/io-handbook/development-styleguide#test-titles-must-be-in-the-form-it-should-

Test titles must be in the form "it should ..."
```suggestion
  it('should respond 404 when the user navigate to the bad endpoint of the local server', async () => {
```

Reply:
ping